packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -391,6 +391,7 @@ export interface CallCompositeStrings {
     complianceBannerTranscriptionStopped: string;
     configurationPageCallDetails?: string;
     configurationPageTitle: string;
+    copyInviteLinkButtonLabel: string;
```

Comment:
Shouldn't be part of the stable api change.
These strings are tied to the call composite PSTN/1:N Calling work